src/Symfony/Component/Notifier/Bridge/Nexmo/Tests/NexmoTransportTest.php

```diff
@@ -13,39 +13,35 @@
 
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\Notifier\Bridge\Nexmo\NexmoTransport;
+use Symfony\Component\Notifier\Bridge\Twilio\TwilioTransport;
 use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Message\ChatMessage;
 use Symfony\Component\Notifier\Message\MessageInterface;
 use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Test\TransportTestCase;
+use Symfony\Component\Notifier\Transport\TransportInterface;
 use Symfony\Contracts\HttpClient\HttpClientInterface;
 
-final class NexmoTransportTest extends TestCase
+final class NexmoTransportTest extends TransportTestCase
```

Comment:
`composer.json` should require the version that contain this class.

Reply:
But `5.1` and bumping composer json notifier -> 5.3 is fine?

Reply:
`5.1.0` -> `5.1.9` does not contains this class, meaning the test with `--prefer-lowest` will fail. You have to change the minimal requirement to `~5.1.10`.
When merging back this path in upper branch, @symfony/mergers have to change `^5.2` into `^5.2.1`

Reply:
Makes sense, thanks for the explanation

Reply:
This PR is introducing classes in a `Test` namespace, which is covered by the BC promise. I believe we should consider this a feature.

Reply:
Indeed. Or we'd better put the new classes in the `Tests/` namespace!

Reply:
Ok, I just followed how its done in Mailer component 👍

Reply:
We could do the same in Mailer btw.
You'll have to list notifier in .github/composer-config.json btw

Reply:
Done

Reply:
> We could do the same in Mailer btw.

PR https://github.com/symfony/symfony/pull/39530

> You'll have to list notifier in .github/composer-config.json bt

done 👍

Reply:
We decided to keep it in `Test/` folder and closed #39530 for Mailer 👍

Reply:
Please target 5.x then, as it is a new feature.

(another option would be to move this into `Tests` in 5.1 and then create a new PR against 5.x to move it into `Test`)

Reply:
But introducing it as _experimental_ and keep in `Test/` would be the same, wouldn't it?
I would like to get this in 5.1, so maintenance of the bridges will be much easier.

cc @nicolas-grekas

Reply:
> (another option would be to move this into Tests in 5.1 and then create a new PR against 5.x to move it into Test)

this option makes sense

Reply:
Ok will do 👍

Reply:
Done